apps/documentation/components/customComponents/Pre.tsx

```diff
@@ -0,0 +1,22 @@
+import CodeCopyButton from './CodeCopyButton';
+
+type PreProps = React.HTMLAttributes<HTMLPreElement> & {
+  children: React.ReactNode;
+  raw?: string;
+};
+
+const Pre = ({ raw, children, ...props }: PreProps) => {
+  return (
+    <pre className="relative my-5 overflow-hidden rounded-xl" {...props}>
+      {children}
+      {raw && (
+        <CodeCopyButton
+          className="absolute right-2 top-2 flex gap-2"
```

Comment:
If the CopyCopyButton component is only ever used here, why not include the class names in the component itself and remove the className prop?

Also: why is there a flex and gap classname here? If this is a style override of the button class names, it should again be included inside the copy button component.

Reply:
You're right. The copy button is being used only once for now. We can adjust the styling if we need to use it somewhere else in the future. Thank you!